### PR TITLE
[Documentation][Customization] Repositories Fix

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/customization/repository.rst [hunk 1]
```diff
@@ -85,11 +85,11 @@ You can form now on use your method in any **Controller**.
 
     <?php
 
-    public function bestsellersAction()
+    public function lowInStockAction()
     {
         $productRepository = $this->container->get('sylius.repository.product');
 
-        $bestsellers = $productRepository->findBySold();
+        $lowInStock = $productRepository->findByOnHold();
```
Comment (lchrusciel) [inline docs/customization/repository.rst]: Then lines above you still have `findBySold`

File: docs/customization/repository.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ You may need for instance:
 How to customize a Repository?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let's assume that you would want to find products that are the bestsellers in your shop.
+Let's assume that you would want to find products that you are running out of in the inventory.
 
 1. Create your own repository class under the ``AppBundle\Repository`` namespace.
```
Comment (lchrusciel) [inline docs/customization/repository.rst]: Should we use our custom debugger? (a few lines below)

File: docs/customization/repository.rst [hunk 3]
```diff
@@ -65,8 +65,8 @@ As a result you will get the ``Sylius\Bundle\CoreBundle\Doctrine\ORM\ProductRepo
 We are using the `Query Builder`_ in the Repositories.
 As we are selecting Products we need to have a join to translations, because they are a translatable resource. Without it in the query results we wouldn't have a name to be displayed.
 
-We are sorting the results by the count of how many times the product has been sold, which is being hold on the ``sold`` field on the specific ``variant`` of each product.
-Then we are limiting the query to 4 by default, to get only 4 bestsellers.
+We are sorting the results by the count of how many products are still available on hand, which is being hold on the ``onHold`` field on the specific ``variant`` of each product.
```
Comment (lchrusciel) [inline docs/customization/repository.rst]: `onHold` -> `onHand`

File: docs/customization/repository.rst (hunk 2 above)
Comment (TheMadeleine) [inline docs/customization/repository.rst]: Is there any significant difference?

Comment (lchrusciel) [inline docs/customization/repository.rst]: No, just mentioning all the time, that we have such a feature ;) Even we are forgetting about it all the time.

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**